collectors/python.d.plugin/postgres/postgres.conf

```diff
@@ -132,3 +136,26 @@ tcpipv6:
     user     : 'postgres'
     host     : '::1'
     port     : 5432
+tcp:
+    name     : 'local'
+    database : 'postgres'
+    user     : 'postgres'
+    passowrd:  'postgres'
+    host     : 'localhost'
+    port     : 5432
+
+tcpipv4:
+    name     : 'local'
+    database : 'postgres'
+    user     : 'postgres'
+    passowrd:  'postgres'    
+    host     : '127.0.0.1'
+    port     : 5432
+
+tcpipv6:
+    name     : 'local'
+    database : 'postgres'
+    user     : 'postgres'
+    passowrd:  'postgres'    
+    host     : '::1'
+    port     : 5432
```

Comment:
We decode configuration in a `dict`, so kyes should be unique.

Adding `password: postgres` to existine tcp* job will do, i think we don't need additional jobs.